pkg/ring/lifecycler.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	minReadyDuration = 1 * time.Minute
+	defaultMinReadyDuration = 1 * time.Minute
```

Comment:
We don't tend to put defaults in consts; we only have them mentioned once in the flag setup.